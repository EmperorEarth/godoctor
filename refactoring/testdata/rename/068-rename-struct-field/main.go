package main

import "fmt"

// Test for renaming a struct field
type myStruct struct {
	Name string // <<<<< rename,7,5,7,5,renamed,pass
}

func main() {
	m := myStruct{Name: "Foo"}
	fmt.Println(m.Name)
}
