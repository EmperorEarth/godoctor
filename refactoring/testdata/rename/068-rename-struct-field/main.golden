package main

import "fmt"

// Test for renaming a struct field
type myStruct struct {
	renamed string // <<<<< rename,7,5,7,5,renamed,pass
}

func main() {
	m := myStruct{renamed: "Foo"}
	fmt.Println(m.renamed)
}
